g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/status/APIResponseStatus.java

```diff
@@ -15,6 +15,7 @@
     public static final Response NO_CONTENT = new Response(204, "No Content");
     public static final Response TRANSLATION_NOT_READY = new Response(205, "Collected resource's translations is not ready");
     public static final Response TRANSLATION_READY = new Response(206, "Collected resource's translations have been ready");
+    public static final Response MULTTRANSLATION_PART_CONTENT = new Response(207, "Part of translations is available");
```

Comment:
Pick another one according to our criterias.

Reply:
this time just use 207, we will uniform the business code in the future